pom.xml

```diff
@@ -220,14 +220,19 @@
       <url>https://git-wip-us.apache.org/repos/asf?p=pdfbox-jbig2.git</url>
       <tag>HEAD</tag>
   </scm>
-	
+
     <dependencies>
-	<dependency>
-		<groupId>junit</groupId>
-		<artifactId>junit</artifactId>
-		<version>4.10</version>
-		<scope>test</scope>
-	</dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.10</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
I am not so sure that it is fine to just add a dependency on Guava here. Why nox fix the existing Implementation by adding weak keys?